### PR TITLE
Companies page

### PR DESCRIPTION
**Pull request by raheraperi**
CompaniesList component lists all companies wrapped in their company weburls
Testing of corresponding RECEIVE_COMPANIES reducer has been tested and passes

### CODE REVIEW

File: client/actions/companies.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ export const requestCompanies = () => {
 export const receiveCompanies = (companies) => {
   return {
     type: RECEIVE_COMPANIES,
-    company: companies.map(company => company)
+    // company: companies.map(company => company)
```
Comment (don-smith) [inline client/actions/companies.js]: remove comment

File: client/components/CompanyList.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      })
+  }
+
+  searchBar (e) {
+    if (e.target.value === '') {
+      this.setState({
+        companyNames: this.props.companyList.map(company => company.name)
+      })
+      this.setState({
+        search: e.target.value
```
Comment (don-smith) [inline client/components/CompanyList.jsx]: move this into the previous `setState`

File: client/components/CompanyList.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    } else {
+      let searchList = this.props.companyList.map(company => company.name)
+      searchList = searchList.filter(res => {
+        return res.toLowerCase().search(e.target.value.toLowerCase()) !== -1
+      })
+      this.setState({
+        search: e.target.value
+      })
+      this.setState({
+        companyNames: searchList
```
Comment (don-smith) [inline client/components/CompanyList.jsx]: move this into the previous `setState`

File: client/components/CompanyList.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      })
+    }
+  }
+
+  updateSearch (e) {
+    this.setState({
```
Comment (don-smith) [inline client/components/CompanyList.jsx]: I don't think you need this function. Remove if you don't need it.

File: client/actions/companies.js (hunk 1 above)
Comment (MichaelMitch) [reply client/actions/companies.js]: Done

File: client/components/CompanyList.jsx (hunk 2 above)
Comment (MichaelMitch) [reply client/components/CompanyList.jsx]: Done

File: client/components/CompanyList.jsx (hunk 4 above)
Comment (MichaelMitch) [reply client/components/CompanyList.jsx]: Done

File: client/components/CompanyList.jsx (hunk 3 above)
Comment (MichaelMitch) [reply client/components/CompanyList.jsx]: Done

**Comment by MichaelMitch** [comment]
All amendments complete

**Pull request merged**